Druid/DRCogitStackToRegisterMappingGenerator.class.st

```diff
@@ -217,15 +221,13 @@ DRCogitStackToRegisterMappingGenerator >> visitCall: aDRCall [
 
 { #category : #visiting }
 DRCogitStackToRegisterMappingGenerator >> visitCogitSendMarshall: aDRCogitSendMarshall [
-
 	"self marshallSendArguments: numArgs."
 
 	generatorMethodBuilder addStatement: (RBMessageNode
 			 receiver: RBVariableNode selfNode
 			 selector: #marshallSendArguments:
 			 arguments:
-			 { (RBLiteralValueNode value:
-				  aDRCogitSendMarshall numberOfArguments) })
+			 { (RBParser parseExpression: aDRCogitSendMarshall numberOfArguments asString) })
```

Comment:
Ok, so here we assume this should always be a jit compile time expression, right?

Reply:
Yes, it works with any expression: Constants or JITExpressions.